ios/FishjamRNClient.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name           = 'MembraneWebRTC'
+  s.name           = 'FishjamRNClient'
```

Comment:
Shouldn't it be just `FishjamClient` like everywhere else?

Reply:
Also, line `15` – could you update the `git` link to https://github.com/fishjam-dev/react-native-client-sdk? As for now it's some random inactive repo

Reply:
Agreed. s.source should be updated.

Reply:
`FishjamClient` is 'reserved' by `ios-client-sdk` pod name. So this name has to be different